libc-test/test/makedev.rs

```diff
@@ -16,11 +16,17 @@ mod t {
 
     extern "C" {
         pub fn makedev_ffi(major: c_uint, minor: c_uint) -> dev_t;
+        pub fn major_ffi(dev: dev_t) -> c_uint;
+        pub fn minor_ffi(dev: dev_t) -> c_uint;
     }
 
     fn compare(major: c_uint, minor: c_uint) {
-        let expected = unsafe { makedev_ffi(major, minor) };
-        assert_eq!(libc::makedev(major, minor), expected);
+        let dev = unsafe { makedev_ffi(major, minor) };
+        assert_eq!(libc::makedev(major, minor), dev);
+        let major = unsafe { major_ffi(dev) };
+        assert_eq!(libc::major(dev), major as _);
+        let minor = unsafe { minor_ffi(dev) };
+        assert_eq!(libc::minor(dev), minor as _);
```

Comment:
Could you name the types here rather than using `_`?

Reply:
I wish I could, but the return type can be `c_int`, `c_uint`, `major_t`, `minor_t` depending on the platform.